recipes/configs/llama3_1/8B_full_single_device.yaml

```diff
@@ -61,7 +62,7 @@ loss:
 max_steps_per_epoch: null
 gradient_accumulation_steps: 1
 optimizer_in_bwd: True
-compile: False # set it to True for better memory and performance
+compile=False  # pytorch compile, set to true for perf/memory improvement# set it to True for better memory and performance
```

Comment:
do you mind quickly double checking? Also, if you are using a script, maybe for a sanity check make just that compile/packed dont appear twice?

Reply:
this should be a colon? `compile: False`

Reply:
Yes, obviously. Fixed